backend/src/packages/files/file.service.ts

```diff
@@ -0,0 +1,92 @@
+import crypto from 'node:crypto';
+
+import { type Service } from '#libs/types/types.js';
+import { FileEntity } from '#packages/files/file.entity.js';
+import { type FileRepository } from '#packages/files/file.repository.js';
+
+import { type AWSService } from './aws.service.js';
+import {
+  type FileGetAllResponseDto,
+  type FileUploadRequestDto,
+  type FileUploadResponseDto,
+} from './libs/types/types.js';
+
+type FileServiceDependencies = {
+  fileRepository: FileRepository;
+  awsService: AWSService;
+};
+
+class FileService implements Service {
+  private fileRepository: FileRepository;
+  private awsService: AWSService;
+
+  public constructor({ fileRepository, awsService }: FileServiceDependencies) {
+    this.fileRepository = fileRepository;
+    this.awsService = awsService;
+  }
+
+  public find(): ReturnType<Service['find']> {
+    return Promise.resolve(null);
+  }
+
+  //FUTURE TODO: CREATE CONTROLLER FOR findById
```

Comment:
let's start comments without FUTURE, just TODO:, and use usual text after that
```suggestion
  // TODO: create controller for findById
```